src/main/java/com/ttbkk/api/domain/hashtag/Hashtag.java

```diff
@@ -0,0 +1,30 @@
+package com.ttbkk.api.domain.hashtag;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+//Entity 클래스
+@Getter
+@NoArgsConstructor
+@Entity
+@Table(name = "hashtag")
+public class Hashtag {
+
+    @Id
```

Comment:
어떤건 첫줄이 비어있고, 어떤건 띄워져있네요
```suggestion
public class Hashtag {
    @Id
```

Reply:
https://github.com/ttbkk-univ/ttbkk-spring/pull/7/commits/c7f0fafe667b6a65ea96724be268255f7465d7f3 수정하였습니다!